eng/pipelines/templates/Tests_On_Mac.yml

```diff
@@ -52,13 +40,5 @@ steps:
       SetCommitStatusForTestResult -PersonalAccessToken $(NuGetLurkerPersonalAccessToken) -VstsPersonalAccessToken $(System.AccessToken) -CommitSha $(Build.SourceVersion) -TestName "$(TestRunDisplayName) On Mac"
     failOnStderr: "true"
   condition: "not(eq(variables['ManualGitHubChecks'], 'false'))"
-
-- task: PublishPipelineArtifact@1
-  displayName: "Publish binlogs"
-  inputs:
-    artifactName: binlog - $(TestRunDisplayName) On Mac - Attempt $(System.JobAttempt)
-    targetPath: $(Build.StagingDirectory)/binlog
-  condition: " or(failed(), eq(variables['System.debug'], 'true')) "
-
 - task: ComponentGovernanceComponentDetection@0
```

Comment:
Can we remove this?  I think its auto-injected now so this one has been redundant for a while.

Reply:
You're correct. I've removed all of them.
https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/sdlanalysis/componentgovernance